### ISSUE TITLE
Strong Customer Authentication

### ISSUE THREAD

**Issue by woolyseal** [bluedynamics/bda.plone.stripe#2]
Did anyone do a Strong Customer Authentication with the bda.plone.stripe AddOn before?

**Comment by agitator** [comment]
Not yet but I have a new shop project right now where I'll be using bda.plone.stripe

Have you already identified what change would be necessary according to https://stripe.com/en-ch/guides/sca-payment-flows#e-commerce or do you need to support a different case?

**Comment by woolyseal** [comment]
I guess the migration guide could help: https://stripe.com/docs/payments/payment-intents/migration

If I understand right there has to be a new version of the python-library (update) and to use for example 3D Secure we have to change the authentication from token source to the new API:

https://stripe.com/docs/payments/payment-methods/transitioning

But i haven't read all of the documentation regarding this yet

**Comment by woolyseal** [comment]
Is there any work in progress? Or can I/we help with this task?

**Comment by jensens** [comment]
AFAIK there is nothing in progress right now, but any help to improve this package is highly appreciated.

**Comment by woolyseal** [comment]
Any news on this topic?

**Comment by jensens** [comment]
> Any news on this topic?

No, currently we are not working on this. We would be happy if you can help with this. If you need any guidance please do not hesitate to ask!

**Comment by woolyseal** [comment]
We are in contact with one of our customers regarding this and hopefully we will get a budget to work on it. I'll keep you posted about this :)

**Comment by woolyseal** [comment]
So, I guess this task is up again. We got some days to work on this. I'll keep you posted.

**Comment by woolyseal** [comment]
Today we will start the test-phase, but I guess it is not a beautiful solution I build there. If the tests are okay, I will make a fork, so you can take a look at the code

**Comment by agitator** [comment]
nice!
why didn't you make a branch here? ... just saying ;-)

**Comment by woolyseal** [comment]
After a really bumpy start this year (personally) I finally got to make a branch. Problem is I can't push it, because it seems I don't have (like I expected) the permissions to do so. And that is the reason why I suggested a fork ;)

**Comment by agitator** [comment]
@wkbkhard welcome back! you should be a member now ;-)

**Comment by woolyseal** [comment]
```
ERROR: Permission to bluedynamics/bda.plone.stripe.git denied to wkbkhard.
fatal: Konnte nicht vom Remote-Repository lesen.

Bitte stellen Sie sicher, dass die korrekten Zugriffsberechtigungen bestehen
und das Repository existiert.
```

May be I am doing something wrong :thinking:

**Comment by jensens** [comment]
The "shop developers team" was not assigned here, I corrected this. Now you should have write access.
